### DISCUSSION TITLE
Thanks for being a legend

### DISCUSSION THREAD

**Discussion by 1creeperbomb** [mm201/pkmnFoundations#85] [General]
Pokemon Classic Network is so underrated it hurts lol. I told all my friends about it and it amazes them that we can easily trade and battle again.

I know you've been working on Game Sync for a while based on your commits. What's the hardest part about it?

I wish I could help but my C# is rusty as hell after working with python and java for so long.

Also anyway average users can help besides just using the network?

**Comment by mm201** [comment]
Thank you for your kind words! The hardest part about Game Sync is figuring out how to format the wake-up payload, i.e. `/dsio/gw?p=savedata.download`. The response has to contain all the goodies you came back with from the Global Link, including items, Pokémon headed to the Entree forest, and Join Avenue visitors. The guy who sells you Dream World furniture may also be involved somehow. If you want to talk more, we can continue this in #8.
